SwiftCompilerSources/Sources/SIL/Location.swift

```diff
@@ -15,6 +15,10 @@ import SILBridging
 public struct Location: Equatable, CustomStringConvertible {
   let bridged: BridgedLocation
 
+  public static var compilerGenerated: Location {
```

Comment:
Is there a better way to avoid a "compiler-generated" location? Compiler generated locations are a problem right now, because they can appear in crash logs and the user has no idea where it comes from.

Reply:
I actually wanted to discuss this as well.

The reason why I had to add this was because while creating the specialized function I need to add instructions into an empty block; these are the "copied" over closures that were originally being passed as arguments.

Currently the Builder does not support adding instructions at the "beginning" of a block. Well it does - by way of the `Builder(atEndOf: someBlock...)` api, but I still need to specify a location there. Would it be correct and perhaps better to use the parent function's SILDebugScope and SILLocation here instead?

Reply:
@eeckstein wanted to check if you had any quick thoughts about this?